.gitlab/deploy_containers/conditions.yml

```diff
@@ -0,0 +1,85 @@
+
+# Rule to make job manual when running deploy pipelines, or automatic and on success on RC pipelines
+.manual_auto_on_rc:
+  - if: $BUCKET_BRANCH != "beta" && $BUCKET_BRANCH != "stable"
```

Comment:
So `nightly` fails here, is it expected?

Reply:
I replicated the existing rules, looks like job having this rule should not be triggered on nightly